src/less/frontend.less

```diff
@@ -301,20 +299,52 @@ body.full-width {
 }
 
 #sidenav-toggle {
-	float: left;
-	@media (max-width: 765px) {
-		padding-left: 15px;
-	}
+	color: white;
+	background-color: #3E3F3A;
+	padding: 3px 5px 1px 5px;
+	border-radius: 5px 5px 0 0;
+	border: none;
+	text-transform: uppercase;
+	font-size: 16px;
+	position: absolute;
+	left: 100%;
+	top: 33%;
+	-webkit-transform: rotate(90deg);
+	-moz-transform: rotate(90deg);
+	-o-transform: rotate(90deg);
+	-ms-transform: rotate(90deg);
+	transform: rotate(90deg);
```

Comment:
Please get rid of the vendor prefixes, transform is pretty standard these days, same thing for transform-origin